README.md

```diff
@@ -14,43 +14,66 @@ Laravel postgis extension
  
  * Geometry functions on the geometry classes (contains(), equals(), distance(), etc… (HELP!))
 
+## Installation
+
+    composer require phaza/laravel-postgis 
+
+Next add the DatabaseServiceProvider to your `config/app.php` file.
+
+    'Phaza\LaravelPostgis\DatabaseServiceProvider',
+
+That's all.
+
 ## Usage
 
-First of all, enable postgis (See note further down)
+A method called enablePostgis() (and disablePostgis()) is included in the Blueprint object.  
+They work on newer postgres installations, but I recommend enabling postgis manually for now.
 
```

Comment:
Sounds good to me. Using those methods in my migrations didn't work as excepted with up and down.